db/schema.rb

```diff
@@ -36,6 +36,21 @@
     t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
   end
 
+  create_table "allocations", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.uuid "from_location_id", null: false
+    t.uuid "to_location_id", null: false
+    t.date "move_date", null: false
```

Comment:
In the FE for moves we use just `date`. Not sure if it's reflected on the BE as well? Anyway, not very important.

Reply:
No worries, happy to change to `date` 👍